pldoc/static/js/color-contrast.js

```diff
@@ -40,12 +39,19 @@ define([
             var bg, fg, ratio, ratios;
 
             $('.example-color').each(function() {
-
-                bg = AccessibilityColorContrast.getL( AccessibilityColorContrast.rgbaToHex( $(this).css('backgroundColor') ) );
-                fg = AccessibilityColorContrast.getL( AccessibilityColorContrast.rgbaToHex( $(this).find('.color-class').css('color') ) );
-
-                    ratio = (Math.max(bg, fg) + 0.05) / (Math.min(bg, fg) + 0.05);
-                    ratios = [4.5, 3]; // 4.5 normal text AA, 3 large text AA
+                bg = AccessibilityColorContrast.getL(
+                        AccessibilityColorContrast.rgbaToHex(
+                            $(this).css('backgroundColor')
+                        )
+                     );
+                fg = AccessibilityColorContrast.getL(
+                        AccessibilityColorContrast.rgbaToHex(
+                            $(this).find('.color-class').css('color')
+                        )
+                     );
```

Comment:
Is this overindented? I would have thought the correct indentation here would be

```javascript
fg = AccessibilityColorContrast.getL(
    AccessibilityColorContrast.rgbaToHex(
        $(this).find('.color-class').css('color')
    )
);
```